File: exercise/sum/solution/correct.java
class Submission {

    public static void main(String[] args) {
        int som = 0;
        for (String arg : args) {
            try {
                som += Integer.parseInt(arg);
            } catch (NumberFormatException e) {
                System.err.println("som: ongeldige argumenten");
                System.exit(1);
            }
        }
        System.out.println(som);

    }
}

File: exercise/sum/solution/correct.kt
import kotlin.system.exitProcess

fun main(args: Array<String>) {
    var som = 0;
    for (arg in args) {
        val num = arg.toIntOrNull();
        if (num == null) {
            System.err.println("som: ongeldige argumenten")
            exitProcess(1)
        }
        som += num
    }
    println(som)
}